modules/fedimint-dummy-client/src/api.rs

```diff
@@ -1,29 +1,28 @@
 use fedimint_core::api::{FederationApiExt, FederationResult, IFederationApi};
+use fedimint_core::epoch::SerdeSignature;
 use fedimint_core::module::ApiRequestErased;
 use fedimint_core::task::{MaybeSend, MaybeSync};
-use fedimint_core::{apply, async_trait_maybe_send, Amount};
-use fedimint_dummy_common::DummyPrintMoneyRequest;
-use secp256k1::XOnlyPublicKey;
+use fedimint_core::{apply, async_trait_maybe_send};
 
 #[apply(async_trait_maybe_send!)]
 pub trait DummyFederationApi {
-    async fn print_money(&self, request: DummyPrintMoneyRequest) -> FederationResult<()>;
+    async fn sign_message(&self, message: String) -> FederationResult<()>;
```

Comment:
Does this module still intend to print money?

Reply:
Yes, we print money by allowing users to submit txs with the fed's private key.

This was a change so that we can print money to another primary module (e.g. ecash).